storage/docker-compose.yml

```diff
@@ -16,6 +16,9 @@ services:
     command: ["bash", "-c", "yarn && yarn start"]
     environment:
       PG_HOST: database
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+      AWS_REGION: ${AWS_REGION}
```

Comment:
bucket name?

Reply:
@petereast ... does the bucket name come from the namespace?

Reply:
/cc @Cooryd Eventually it'll be derived from the config but at the moment it's hard coded. It's probably worth knowing that the namespace we're using is `someNameSpace` and the bucket we're using is `storage-service-demo`

Reply:
@giorgiosironi - you happy with this or I can make these pull from env's too as part of this PR ?

Reply:
as long as we know the name of the bucket, it can be created

Reply:
Yeah, I'm happy, the bucket name is hard-coded to `storage-service-demo` - I'm going to merge this now